sql/upgrade/sl2.8-1.5.sql

```diff
@@ -458,18 +458,29 @@ ALTER TABLE sl28.acc_trans ADD COLUMN lsmb_entry_id integer;
 update sl28.acc_trans
   set lsmb_entry_id = nextval('acc_trans_entry_id_seq');
 
-INSERT INTO acc_trans
-(entry_id, trans_id, chart_id, amount, transdate, source, cleared, fx_transaction,
-        memo, approved, cleared_on, voucher_id)
-SELECT lsmb_entry_id, trans_id, (select id
-                    from account
-                   where accno = (select accno
-                                    from sl28.chart
-                                   where chart.id = acc_trans.chart_id)),
-                                    amount, transdate, source,
-        CASE WHEN cleared IS NOT NULL THEN TRUE ELSE FALSE END, fx_transaction,
-        memo, approved, cleared, vr_id
-   FROM sl28.acc_trans
+INSERT INTO invoice (id, trans_id, parts_id, description, qty, allocated,
+           sellprice, fxsellprice, discount, assemblyitem, unit,
+           deliverydate, serialnumber)
+   SELECT  id, trans_id, parts_id, description, qty, allocated,
+           sellprice, fxsellprice, discount, assemblyitem, unit,
+           deliverydate, serialnumber
+      FROM sl28.invoice;
+
+INSERT INTO acc_trans (entry_id, trans_id, chart_id, amount, transdate,
+                       source, cleared, fx_transaction,
+                       memo, approved, cleared_on, voucher_id, invoice_id)
+     SELECT lsmb_entry_id, trans_id,
+            (select id
+               from account
+              where accno = (select accno
+                               from sl28.chart
+                              where chart.id = acc_trans.chart_id)),
+            amount, transdate, source,
+            CASE WHEN cleared IS NOT NULL THEN TRUE ELSE FALSE END,
+            fx_transaction,
+            memo, approved, cleared, vr_id, invoice.id
+       FROM sl28.acc_trans
+  LEFT JOIN sl28.invoice ON invoice.id = acc_trans.id
   WHERE chart_id IS NOT NULL
         AND trans_id IN (SELECT id FROM transactions);
 
```

Comment:
Which trans_id are you referring to? Both invoice and acc_trans have that field to the query is bound to be ambiguous at runtime.